DataHub/UserInterface/02e_Channels.md

```diff
@@ -0,0 +1,144 @@
+# Channels
+
+*DataHub > Settings > Tab CHANNELS*
+
+![Channels](../../Assets/Screenshots/DataHub/Settings/Channels/ChannelList.png "[Channels]")
+
+**Channel list**
+
+- ![Search](../../Assets/Icons/Search.png "[Search]") (Search)   
+  Click this button to display the search bar and search for a channel.
+
+- ![Refresh](../../Assets/Icons/Refresh01.png "[Refresh]") (Refresh)   
+  Click this button to update the list of channels.
+
+- ![Columns](../../Assets/Icons/Columns.png "[Columns]") Columns (x)   
+  Click this button to display the columns bar and customize the displayed columns and the order of columns in the list. The *x* indicates the number of columns that are currently displayed in the list.
+
+- ![Filter](../../Assets/Icons/Filter.png "[Filter]") Filter (x)   
+  Click this button to display the filter bar and customize the active filters. The *x* indicates the number of filters that are currently active.
+
+- [x]     
+  Select the checkbox to display the editing toolbar. If you click the checkbox in the header, all channels in the list are selected.
+
+- [EDIT]   
+  Click this button to edit the selected channel. This button is only displayed if a single checkbox of a channel is selected. Alternatively, you can click directly a row in the list to edit a channel.
```

Comment:
```suggestion
  Click this button to edit the selected channel. This button is only displayed if the checkbox of a channel is selected. Alternatively, you can click directly a row in the list to edit a channel.
```
Vgl. 02_ETL.md + Default sentences: This button is only displayed if the checkbox of an attribute set mapping is selected / This button is only displayed if a single checkbox in the list of XYZ is selected. 
Ggf. Default sentences aktualisieren.